src/main/java/org/spongepowered/common/config/category/CollisionModCategory.java

```diff
@@ -34,17 +34,17 @@
 public class CollisionModCategory extends ConfigCategory {
 
     @Setting(value = "enabled", comment = "If 'false', entity collision rules for this mod will be ignored.")
-    private boolean isEnabled = true;
+    private final boolean isEnabled = true;
```

Comment:
Undo these final modifiers on `@Setting` fields, Configurate's object mapping will not work with these change.

Reply:
Fighting the auto formatter ... Done.